policy/modules/kernel/files.if

```diff
@@ -1844,6 +1844,24 @@ interface(`files_dontaudit_list_all_mountpoints',`
 	dontaudit $1 mountpoint:dir list_dir_perms;
 ')
 
+########################################
+## <summary>
+##	Watch all mountpoints.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`files_watch_all_mountpoints',`
+	gen_require(`
+		attribute mountpoint;
+	')
+
+	allow $1 mountpoint:dir { watch_mount watch_with_perm };
```

Comment:
Need two interfaces. Also "watch" in the interface name would refer to the dir:watch permission, so we need new verbs here.

Reply:
I will setup two interfaces and try to make names that are reasonable, but I'm open to naming suggestions.